src/test/java/teammates/test/cases/ui/browsertests/InstructorFeedbackEditPageUiTest.java

```diff
@@ -288,709 +259,69 @@ private void testGetQuestionLink() {
-
-        feedbackEditPage.verifyHtmlMainContent("/instructorFeedbackNumScaleQuestionEditSuccess.html");
-    }
-
-    private void testDeleteNumScaleQuestionAction() {
-        ______TS("NUMSCALE: qn delete then cancel");
-
-        feedbackEditPage.clickAndCancel(feedbackEditPage.getDeleteQuestionLink(2));
-        assertNotNull(BackDoor.getFeedbackQuestion(courseId, feedbackSessionName, 2));
-
-        ______TS("NUMSCALE: qn delete then accept");
-
-        feedbackEditPage.clickAndConfirm(feedbackEditPage.getDeleteQuestionLink(2));
-        assertEquals(Const.StatusMessages.FEEDBACK_QUESTION_DELETED, feedbackEditPage.getStatus());
-        assertNull(BackDoor.getFeedbackQuestion(courseId, feedbackSessionName, 2));
     }
-    
-    @SuppressWarnings("unused")
-    private void ____CONSTSUM_OPTION_methods___________________________________() {
-    }
-    
-    private void testNewConstSumOptionQuestionFrame() {
-        ______TS("CONSTSUM-option: new question (frame) link");
 
-        feedbackEditPage.selectNewQuestionType("Distribute points (among options) question");
-        feedbackEditPage.clickNewQuestionButton();
-        assertTrue(feedbackEditPage.verifyNewConstSumQuestionFormIsDisplayed());
-    }
-    
-    private void testInputValidationForConstSumOptionQuestion() {
-        
-        ______TS("empty options");
-        
-        feedbackEditPage.fillQuestionBox("ConstSum-option qn");
-        feedbackEditPage.fillConstSumPointsBox("", -1);
-        
-        assertEquals("1", feedbackEditPage.getConstSumPointsBox(-1));
-        
-        feedbackEditPage.clickAddQuestionButton();
+    private void testCopyQuestion() {
         
-        assertEquals("Too little options for Distribute points (among options) question. Minimum number of options is: 2.", feedbackEditPage.getStatus());
+        ______TS("Success case: copy questions successfully");
         
-        ______TS("remove when 1 left");
-
-        feedbackEditPage.selectNewQuestionType("Distribute points (among options) question");
-        feedbackEditPage.clickNewQuestionButton();
-        feedbackEditPage.fillQuestionBox("Test const sum question");
-        assertTrue(feedbackEditPage.verifyNewConstSumQuestionFormIsDisplayed());
-
-        feedbackEditPage.clickRemoveConstSumOptionLink(1, -1);
-        assertEquals(false, feedbackEditPage.isElementPresent("constSumOptionRow-1--1"));
-
-        // TODO: Check that after deleting, the value is cleared
-        assertEquals(true, feedbackEditPage.isElementPresent("constSumOptionRow-0--1"));
-        feedbackEditPage.clickRemoveConstSumOptionLink(0, -1);
-        assertEquals(true, feedbackEditPage.isElementPresent("constSumOptionRow-0--1"));
-        feedbackEditPage.clickAddQuestionButton();
-        assertEquals("Too little options for Distribute points (among options) question. Minimum number of options is: 2.", feedbackEditPage.getStatus());
+        ThreadHelper.waitFor(1000);
```

Comment:
We should have a separate issue to look for waits in tests that use absolute values, and change them to wait for some element if possible.